v4/weaviate/graphql/neartextbuilder.go

```diff
@@ -1,11 +1,72 @@
 package graphql
 
 import (
-	"encoding/json"
 	"fmt"
 	"strings"
 )
 
+// fldMover is a type representing field names of a move sub query
+type fldMover string
+
+const (
+	fldMoverConcepts fldMover = "concepts"
+	fldMoverForce    fldMover = "force"
+	fldMoverID       fldMover = "id"
+	fldMoverBeacon   fldMover = "beacon"
+	fldMoverObjects  fldMover = "objects"
+)
+
+// MoveParameters to fine tune Explore queries
+type MoveParameters struct {
+	// Concepts that should be used as base for the movement operation
+	Concepts []string
+	// Force to be applied in the movement operation
+	Force float32
+	// Movers is a list of movers used to adjust the serach direction
+	Movers []MoverObject
+}
+
+func (m *MoveParameters) String() string {
+	concepts := marshalStrings(m.Concepts)
+	ms := make([]string, 0, len(m.Movers))
+	for _, m := range m.Movers {
+		if s := m.String(); s != EmptyObjectStr {
+			ms = append(ms, m.String())
+		}
+	}
+	if len(ms) < 1 {
```

Comment:
There are quite a lot of if statements that are testing certain scenarios, which I think can be ommited, you can just build the parts of the argument (concepts, force, objects) and then just at the very end concatenate the results into one string as we do this in other argument builders

Reply:
I am not quite sure what can be omitted. 

However the purpose of the statement above is to exclude the moverObjects when there is no mover specified